src/amo/reducers/featured.js

```diff
@@ -0,0 +1,33 @@
+import {
+  FEATURED_GET,
+  FEATURED_LOADED,
+  FEATURED_FAILED,
+} from 'core/constants';
+
+const initialState = {
+  count: 0,
+  loading: false,
+  results: [],
```

Comment:
That actually works for me, but I worried about too tightly coupling things that could maybe be used separately. I think you're right though, and one reducer could be used. I think I got a bit tripped up using `Promise.all()` and assigning things to the right state but maybe I need to give it another go.

I'll try combining them all and making the tweaks suggested and see how it goes. Thanks 👍